app/views/filters/_form.html.erb

```diff
@@ -42,9 +42,16 @@
           </span>
       </div>
 
-      <div id="granular_form" style="<%= !f.object.granular? ? 'display:none' : '' %>">
+      <div id="override_taxonomy_form" class="<%= f.object.allows_taxonomies_filtering? ? '' : 'hidden' %>">
+        <%= checkbox_f f, :override?,
+                       :help_inline => _("Filters inherit organizations and locations from their role by default. If override field is enabled, filter can override the set of its organizations and locations. Later role changes will not affect such filter. After disabling override field, role organizations and locations apply again."),
```

Comment:
Do this through a `popover('', _('filter inherit...` , in order to use the Field Level Help pattern a bit better (it clarifies the form a lot) (https://www.patternfly.org/patterns/field-level-help/) . There are examples in app/views/nic